src/amo/reducers/collections.js

```diff
@@ -315,14 +316,15 @@ type LoadCurrentCollectionPageAction = {|
 
 export const loadCurrentCollectionPage = ({
   addons,
+  numberOfAddons,
 }: LoadCurrentCollectionPageParams = {}): LoadCurrentCollectionPageAction => {
-  if (!addons) {
-    throw new Error('addons are required');
-  }
+
+  invariant(addons, 'The addons parameter is required');
+  invariant(numberOfAddons, 'The numberOfAddons parameter is required');
```

Comment:
This looks good, but `0` is a valid value for `numberOfAddons`, and this code will fail with a value of `0`, which is why I suggested earlier that you could either check that `numberOfAddons` is neither `undefined` or `null`, or just check that it's a number.

Reply:
oh sorry.
Please check again. Thank you.